src/components/ContextSubMenu.js

```diff
@@ -3,27 +3,37 @@ import PropTypes from 'prop-types';
 import { Link } from 'gatsby';
 import { SprkLink } from '@sparkdesignsystem/spark-react';
 
-function ContextSubMenu({heading, collection, directory}) {
+function ContextSubMenu({ heading, collection, directory }) {
+  const ProcessedSublinks = collection.filter(
+    (item) => item.node.parent.name !== 'guides',
+  );
```

Comment:
Needed to process through collection so the menu doesn't look like this:

Guides
  Guides
  Validation Messages